internal/rest/client.go

```diff
@@ -342,6 +343,13 @@ func (c *Client) LastError() error {
 	return c.lastErr
 }
 
+// LastErrTime returns previous error time
```

Comment:
We can bring back the previous change now since we have made safer changes at the caller side.